src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -215,6 +215,14 @@ bool AsyncRPCOperation_sendmany::main_impl() {
     CAmount selectedUTXOAmount = 0;
     bool selectedUTXOCoinbase = false;
     if (isfromtaddr_) {
+        // Get dust threshold
+        CKey secret;
+        secret.MakeNewKey(true);
+        CScript scriptPubKey = GetScriptForDestination(secret.GetPubKey().GetID());
+        CTxOut out(CAmount(1), scriptPubKey);
+        CAmount dustThreshold = out.GetDustThreshold(minRelayTxFee);
+        CAmount dustChange = -1;
```

Comment:
I kind of see what you mean but I deliberately named it `dustChange` so its obvious this code block is looking/checking for change which is dust.

Once the UTXO has been selected, if you scroll down, you'll see that `change` is set at the beginning of two scenarios which send from taddr.